client/ProductSizeButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function ProductSizeButton(props) {
```

Comment:
My linter is telling me it wants to see function expressions, not function declarations. It's an eslint rule in the hack reactor eslint config

Reply:
Thanks, I think maybe my HR esLint is maybe not installed. I was following component examples from ReactJS, so it's odd that HR has an issue with it. I'll make the change!